test/service/test_request_builder.py

```diff
@@ -229,14 +230,8 @@ def test_create_request_missing_values(self):
                         {
                             "constant_score": {
                                 "filter": {
-                                    "bool": {
-                                        "must_not": [
-                                            {
-                                                "exists": {
-                                                    "field": "entity_id.keyword"
-                                                }
-                                            }
-                                        ]
+                                    "terms": {
+                                        "entity_id.keyword": None
```

Comment:
This looks fishy. It should be a valid ES query not an intermediate form. Instead of adjusting the assertion you should modify the test code so it applies the transformation to the final form.

Reply:
Still smells fishy.